### PR TITLE
Added CPF and CNPJ in pt_BR provider

### PR DESCRIPTION
**Pull request by acfreitas**

### CODE REVIEW

File: src/Faker/Provider/Imei.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Faker\Provider;
+
+class Imei
+{
+
+    /**
+     * International Mobile Equipment Identity (IMEI)
```
Comment (fzaninotto) [inline src/Faker/Provider/Imei.php]: Should not be part of this PR, please remove

**Comment by fzaninotto** [comment]
#545 already does that, duplicate.

**Comment by acfreitas** [comment]
Sorry, I created a new PR to IMEI.
